modules/world/evaluation/evaluator_speed_advantage_overtake.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2020 fortiss GmbH
+//
+// This work is licensed under the terms of the MIT license.
+// For a copy, see <https://opensource.org/licenses/MIT>.
+
+#include "modules/world/evaluation/evaluator_speed_advantage_overtake.hpp"
+
+#include "modules/world/evaluation/labels/behind_of_label_function.hpp"
+#include "modules/world/evaluation/labels/front_of_label_function.hpp"
+#include "modules/world/evaluation/labels/left_of_label_function.hpp"
+#include "modules/world/evaluation/labels/rel_speed_label_function.hpp"
+#include "modules/world/evaluation/labels/right_of_label_function.hpp"
+
+namespace modules {
+namespace world {
+namespace evaluation {
+const char EvaluatorSpeedAdvantageOvertake::formula_[] =
```

Comment:
add description or link to paper

Reply:
we will provide a link once the paper got accepted. add some small doc string, then it should be okay for now.